npm-shrinkwrap.json

```diff
@@ -38,13 +38,11 @@
       "version": "4.11.5",
       "from": "ajv@>=4.9.1 <5.0.0",
       "resolved": "https://registry.npmjs.org/ajv/-/ajv-4.11.5.tgz",
-      "dev": true,
       "dependencies": {
         "json-stable-stringify": {
           "version": "1.0.1",
           "from": "json-stable-stringify@>=1.0.1 <2.0.0",
-          "resolved": "https://registry.npmjs.org/json-stable-stringify/-/json-stable-stringify-1.0.1.tgz",
-          "dev": true
+          "resolved": "https://registry.npmjs.org/json-stable-stringify/-/json-stable-stringify-1.0.1.tgz"
```

Comment:
Haha, I'm using the new npm version already. In this particular case, `json-stable-stringify` was required by a development dependency, but after we installed `request`, `json-stable-stringify` is now also a production dependency, which caused the `dev: true` to go away.